src/error_types/utils.ts

```diff
@@ -0,0 +1,44 @@
+import { HTTP_ERROR, HttpErrorInfo } from "./http_errors"
+
+// Input: Error code that can be either a status_code or an http_code
+// Output: HttpErrorInfo object containing the correct values for given code
+export const formatErrorCode = (code: any) => {
+  // Default to 500 internal server error
+  let formattedErrorInfo: HttpErrorInfo = {
+    status: HTTP_ERROR.internal.status,
+    code: HTTP_ERROR.internal.code,
+    description: `${HTTP_ERROR.internal.description} Error Format util`,
+  }
+
+  // if input is a number we want to find the corresponding error code and set
+  // the values based on that code
+  if (!isNaN(+code)) { // if e.code is not not a number; if e.code is a number
+    Object.values(HTTP_ERROR).forEach((errorType) => {
+      // if code is a number, set default code in case we haven't defined that specific code
+      formattedErrorInfo = {...formattedErrorInfo, code: +code}
```

Comment:
With this logic - the code should always be set to the last touched errorType that we find in the forEach loop called on HTTP_ERROR values.  I've already implemented something similar here: https://github.com/looker-open-source/actions/blob/master/src/hub/action_response.ts#L22